c_sharp/image_classification/model/augment_model.py

```diff
@@ -0,0 +1,20 @@
+import onnx
+import torch
+from onnxruntime_extensions import pnp
+
+mobilenet_model = onnx.load_model('mobilenetv2-12.onnx')
+augmented_model = pnp.SequentialProcessingModule(
```

Comment:
Where does this model come from? I wasn't part of the PR (binary files need to be manually added).